scripts/ci/jobs/gke_postgres_upgrade_tests.py

```diff
@@ -10,6 +10,8 @@
 from ci_tests import PostgresUpgradeTest
 from post_tests import PostClusterTest, FinalPost
 
+os.environ["ROX_POSTGRES_DATASTORE"] = "true"
```

Comment:
This shouldn't be needed because of what this tests does and how it does it.  This test actually starts in Rocks and goes back and forth.  I'm not even certain setting this would make a difference.

Reply:
Yup, it is not, it seemed really odd that gke_upgrade_tests.py explicitly turns the flag off. But the postgres counterpart doesn't do anything about that flag. In the least, a comment here would help. To keep the test scripts less "magical".